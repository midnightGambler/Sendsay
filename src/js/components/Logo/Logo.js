import React from "react";

const Logo = () => {
  return (
    <div className="logo logo-mb">
      <div className="logo__ellipse logo__ellipse-mr" />
      <div className="logo__rectangle logo__rectangle-mr" />
      <div className="logo__ellipse" />
      <div className="logo__rectangle-flip">
        <div className='logo__rectangle-flip--inner' />
      </div>
    </div>
  );
};

export default Logo;
