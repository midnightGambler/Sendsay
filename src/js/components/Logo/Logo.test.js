import React from "react";
import { shallow } from "enzyme";
import Logo from "./Logo";

it("should render component properly", () => {
  const logo = shallow(<Logo />);
  expect(logo).toMatchSnapshot();
});
