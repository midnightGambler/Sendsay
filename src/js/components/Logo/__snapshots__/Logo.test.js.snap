// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component properly 1`] = `
<div
  className="logo logo-mb"
>
  <div
    className="logo__ellipse logo__ellipse-mr"
  />
  <div
    className="logo__rectangle logo__rectangle-mr"
  />
  <div
    className="logo__ellipse"
  />
  <div
    className="logo__rectangle-flip"
  >
    <div
      className="logo__rectangle-flip--inner"
    />
  </div>
</div>
`;
